dev-tools/omicron-dev/src/main.rs

```diff
@@ -78,6 +79,18 @@ impl RunAllArgs {
         >(&mut config, 0)
         .await
         .context("error setting up services")?;
+
+        println!("omicron-dev: Adding disks to first sled agent");
+
+        // This is how our integration tests are identifying that "disks exist"
+        // within the database.
+        //
+        // This inserts:
+        // - DEFAULT_ZPOOL_COUNT zpools, each of which contains:
+        //   - A crucible dataset
+        //   - A debug dataset
+        DiskTest::new(&cptestctx).await;
```

Comment:
As an example of where this is useful:

https://github.com/oxidecomputer/omicron/pull/7972

We can create (and store) support bundles on the simulated system.